src/vivarium/framework/population/manager.py

```diff
@@ -682,6 +683,10 @@ def extract_columns_from_query(self, query: str) -> set[str]:
         query = re.sub(r"\s+", " ", query).strip()
         return set(query.split(" ") + columns)
 
+    def add_tracked_query(self, query: str) -> str:
+        """Combines the provided query with all registered tracked queries."""
+        return combine_queries(self.tracked_queries, query)
```

Comment:
Nit: I prefer to import modules rather than functions, so that it's easy to see that this is an imported function rather than one actually in this module.